tests/test_cases/shutdown.py

```diff
@@ -0,0 +1,10 @@
+#!/bin/python2
+# Checks if any failure occurs
+import sys
+
+lind_result = sys.argv[1]
+host_result = sys.argv[2]
+
```

Comment:
it would be better if this tested for correctness more than just checking for failure

Reply:
Well if the shutdown affects other things its a problem as well.